admin/metabox/class-metabox-keyword-synonyms-button.php

```diff
@@ -17,10 +17,6 @@ class WPSEO_Metabox_Keyword_Synonyms_Button {
 	 */
 	public function get_link() {
 
-		if ( WPSEO_UTILS::is_yoast_seo_premium() ) {
```

Comment:
Ah, didn't notice this check was already here. :-) But indeed better to move it to the place where this is consumed!